rest-service/setup.py

```diff
@@ -39,6 +38,7 @@
     'cachetools>=3,<4',
     'email-validator>1,<2',
     'retrying',
+    'pydantic',
```

Comment:
We maybe want to pin this to make sure it doesn't jump to the next major version arbitrarily?

Reply:
I think it's fine, we do have a concrete version in reqs.txt, so this only makes for "how hard is it to upgrade". Things like pydantic should be... very easy, on major versions

Reply:
Fair point, request withdrawn!